lib/provider/volume_manager.go

```diff
@@ -38,9 +38,8 @@ type VolumeManager interface {
 	// details by usig user provided volume name
 	GetVolumeByName(name string) (*Volume, error)
 
-	// Others
-	// Get volume lists by using snapshot tags
-	ListVolumes(tags map[string]string) ([]*Volume, error)
+	// Get volume lists by using filters
+	ListVolumes(limit int, start string, tags map[string]string) (*VolumeList, error)
```

Comment:
you need to add this change in the fake implementation as well

Reply:
Already added.
https://github.com/IBM/ibmcloud-storage-volume-lib/pull/202/files#diff-f1aeff95975b31a090ab38c466347457L235